validation/markets-validator.js

```diff
@@ -656,7 +660,18 @@ function MetadataValidator(metadata, cityName) {
                     }
                 }
             });
-            request.on('error', handleError);
+            request.on('error', function(error) {
+                if (error.code === "UNABLE_TO_VERIFY_LEAF_SIGNATURE") { // München
```

Comment:
An alternative (no recommendation from my end) could be to download the cert and set an environment variable:

https://github.com/node-fetch/node-fetch/discussions/1702#discussioncomment-7324457